privacyidea/lib/tokens/certificatetoken.py

```diff
@@ -334,3 +339,79 @@ def revoke(self):
         log.info("CRL {0!s} created.".format(crl))
 
         return revoked
+    
+    
+    @staticmethod
+    def api_endpoint(request, g):
+        """
+        Author : Tarek EL ALLAM
+        This provides a function to be plugged into the API endpoint
+        /ttype/<tokentype> which is defined in api/ttype.py
+        See :ref:`rest_ttype`.
+        Desc : 
+        Add API call in order to get the last generated certificate assigned, activated, unrevoked and unlocked.
```

Comment:
actually we already have such an endpoint, it is the ``GET /token/`` endpoint, which needs an administrator or a user to authenticate. But you actually can have the same result.
This approach leaves the information, which certificate the user has, open to the world without any authentication.
Why would you do this?
Would you be able to have a service account to receive this information?